browser/brave_browser_process_impl.cc

```diff
@@ -251,3 +260,12 @@ brave::BraveStatsUpdater* BraveBrowserProcessImpl::brave_stats_updater() {
     brave_stats_updater_ = brave::BraveStatsUpdaterFactory(local_state());
   return brave_stats_updater_.get();
 }
+
+void BraveBrowserProcessImpl::CreateProfileManager() {
```

Comment:
can we avoid moving this here? It makes git blame more confusing

Reply:
I think it's better to have it this way to have a natural order of methods between *.h and *.cc

Reply:
that's fine, but those changes are better made in separate PRs to avoid confusing things